reference/twig_reference.rst

```diff
@@ -371,18 +372,22 @@ absolute_url
 ``path``
     **type**: ``string``
 
-Returns the absolute URL for the given absolute path. This is useful to convert
-an existing path:
+Returns the absolute URL from the passed relative path. Assume the following
+code was used to render ``http://example.com/sub/list.html``:
```

Comment:
I've re-read your code, and it makes sense now. I'd re-word the second sentence:

```rst
For example, assume you're on the following page in your app
``http://example.com/products/hover-board``
```

I think the `.html` in the URL confused me for some reason too :). Obviously, if you change to `/products`, the code below will need to change too